recipes/dcase2021_task4_baseline/local/sepsed_trainer.py

```diff
@@ -0,0 +1,725 @@
+import os
+import random
+from copy import deepcopy
+from pathlib import Path
+
+import pandas as pd
+import pytorch_lightning as pl
+import torch
+from torchaudio.transforms import AmplitudeToDB, MelSpectrogram
+
+from desed_task.data_augm import mixup
+from desed_task.utils.scaler import TorchScaler
+import numpy as np
+
+from .utils import (
+    batched_decode_preds,
+    log_sedeval_metrics,
+)
+from desed_task.evaluation.evaluation_measures import (
+    compute_per_intersection_macro_f1,
+    compute_psds_from_operating_points,
+)
+
+
+class SEPSEDTask4_2021(pl.LightningModule):
```

Comment:
Maybe this class could have been a subclass of `SEDTask4_2021`, so it is easier to see the differences ? (I'm not sure it's a good idea yet)
By checking, it seems only the "detect" function is added and changes the way we do it, right ?

Reply:
There also the paths which are different. self.hparams["data"]["weak_folder_sep"] vs hparams["data"]["weak_folder"].-
Probably I can change the paths to be the same

Reply:
Indeed, then we have something easy to understand when going from one to another